npm/test/basics.js

```diff
@@ -879,3 +880,251 @@ test("debug service compiling multiple sources - web worker", async () => {
     debugService.terminate();
   }
 });
+
+test("CreateIntegerTicks: invalid inputs", () => {
+  runAndAssertIntegerTicks(2, 1, []);
+  runAndAssertIntegerTicks(0, 2, []);
+  runAndAssertIntegerTicks(-5, 100, []);
+});
+
+test("CreateIntegerTicks: below 100", () => {
+  runAndAssertIntegerTicks(1, 1, [1]);
+  runAndAssertIntegerTicks(3, 3, [3]);
+  runAndAssertIntegerTicks(4, 6, [4, 5, 6]);
+  runAndAssertIntegerTicks(1, 100, [1, 10, 100]);
+  runAndAssertIntegerTicks(1, 10, [1, 10]);
+  runAndAssertIntegerTicks(1, 9, [1]);
+  runAndAssertIntegerTicks(2, 10, [10]);
+  runAndAssertIntegerTicks(2, 9, [2, 3, 4, 5, 6, 7, 8, 9]);
```

Comment:
Am I reading the above tests right that 
 - if my data runs 1 to 9, I just get a tick at 1
 - if my data runs 2 to 10, I just get a tick at 10
 - if my data runs 2 to 9, I get 8 ticks from 2 through 9

Maybe that 'feels' right in practice, but seems a little odd.

Reply:
Thank you, @billti! The intention was to show 1, 10, 100, ... if possible. If at least one of them is shown, then do not show anything else. Otherwise, show intermediate results on the lower level, e.g. 2, 3, 4, 5; or 50, 60, 70, 80, 90; or 34, 35, 36 etc.
Yes, in the "worst" case it could be 2 through 9 or similar 8 numbers. Could be worse in case of 1, 10, ... 1'000'000'000, etc.

So, the number of ticks could be from 1 to 8. My original goal was to find at least a single tick. Everything else was secondary. Maybe we could set more elegant goals like to find from 2 to 5 ticks or so. Thoughts?

Reply:
Let us consider this aside from the current PR. I have created https://github.com/microsoft/qsharp/issues/1036